pyanaconda/display.py

```diff
@@ -84,25 +84,6 @@ def start_user_systemd():
     os.environ["DBUS_SESSION_BUS_ADDRESS"] = session_bus_address
     log.info("The session bus address is set to %s.", session_bus_address)
 
-# Spice
-
-def start_spice_vd_agent():
```

Comment:
Yeah, this is a bit strange - we looked at this with @jstodola and it looks like copy-paste no longer works to VMs (at least in virt manager) but works fine when connecting via an RDP client. I find this strange, as I was assuming this piece of code was taking care of copy-paste working in all environments.

In any case, definitely something to look into in the future, as copy-past is often very convenient.